src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -60,6 +60,15 @@ namespace traits
         type(adios_unsigned_long) {}
     };
 
+    template<>
+    struct PICToAdios<uint64_cu>
```

Comment:
above: I think our current implementation is not ideal, we should do
```C++
/** Trait for unsigned long */
     template<>
     struct PICToAdios<unsigned long>
     {
         ADIOS_DATATYPES type;
         
         PICToAdios() :
         type(adios_unsigned_long) {}
     };
```

since adios does not identify 64 and 32bit directly for ints.